From 32c92435ffe0e7caf581459af6448894cf6fa8f0 Mon Sep 17 00:00:00 2001
From: Pierre-Yves Luyten <py@luyten.fr>
Date: Wed, 8 Jul 2015 00:50:15 +0200
Subject: note-view : on color changed : refer to emiting note

Instead of using priv->note, which seems in some cases to be broken.
See RH BZ #1204815
---
 src/bjb-note-view.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/bjb-note-view.c b/src/bjb-note-view.c
index 918a508..bcacd0c 100644
--- a/src/bjb-note-view.c
+++ b/src/bjb-note-view.c
@@ -176,7 +176,7 @@ on_note_color_changed_cb (BijiNoteObj *note, BjbNoteView *self)
 
   g_return_if_fail (BIJI_IS_NOTE_OBJ (note));
 
-  biji_note_obj_get_rgba (priv->note, &color);
+  biji_note_obj_get_rgba (note, &color);
   if (color.red < 0.5)
     font_color = "white";
   else
@@ -189,7 +189,7 @@ on_note_color_changed_cb (BijiNoteObj *note, BjbNoteView *self)
    */
   text = g_strdup_printf (_("<b>Last updated</b> %s"),
                           biji_note_obj_get_last_change_date_string
-			            (priv->note));
+			            (note));
   span = g_strdup_printf ("<span color='%s'>%s</span>", font_color, text);
   gtk_label_set_markup (GTK_LABEL (priv->last_update), span);
 
-- 
cgit v0.12

